wiki/Split_large_file.md

```diff
@@ -40,84 +40,83 @@ def batch_iterator(iterator, batch_size) :
     entries from the supplied iterator.  Each list will have
     batch_size entries, although the final list may be shorter.
     """
-    entry = True #Make sure we loop once
-    while entry :
+    entry = True  # Make sure we loop once
+    while entry:
         batch = []
-        while len(batch) < batch_size :
-            try :
+        while len(batch) < batch_size:
+            try:
                 entry = iterator.next()
-            except StopIteration :
+            except StopIteration:
                 entry = None
-            if entry is None :
-                #End of file
+            if entry is None:
+                # End of file
                 break
             batch.append(entry)
-        if batch :
+        if batch:
             yield batch
 ```
 
 Here is an example using this function to divide up a large FASTQ file,
-SRR014849.fastq (from this [compressed
-file](ftp://ftp.ncbi.nlm.nih.gov/sra/static/SRX003/SRX003639/SRR014849.fastq.gz)
-at the NCBI):
+`SRR014849_1.fastq` (from this [compressed
```

Comment:
I've not double checked what is in `SRR014849.fastq` versus `SRR014849_1.fastq` and `SRR014849_2.fastq` but it doesn't really matter for this example - as long as we're consistent.